Gitea/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "author": {
+        "name": "CRUGG",
+        "id": "228965621478588416"
+    },
+    "service": "Gitea",
+    "description": {
+        "en": "Gitea is an Open-Source Git Platform heavily inspired by GitHub",
+        "de": "Gitea ist eine Open-Source Git Plattform, welche sehr stark an GitHub angelehnt ist"
+    },
+    "url": [
+        "gitea.com"
+    ],

```

Comment:
if there is only 1 url it shouldn't be in an array

Reply:
I did that because in the very near future I also want to support their homepage gitea.io. I can also change it for now though.

Reply:
For now you should change it.